packages/apm/src/integrations/tracing.ts

```diff
@@ -14,6 +14,73 @@ import { Span as SpanClass } from '../span';
 import { SpanStatus } from '../spanstatus';
 import { Transaction } from '../transaction';
 
+/** Holds the latest LargestContentfulPaint value (it changes during page load). */
+let lcp: { [key: string]: any };
+
+/** Force any pending LargestContentfulPaint records to be dispatched. */
+let forceLCP = () => {
+  /* No-op, replaced later if LCP API is available. */
+};
+
+// Based on reference implementation from https://web.dev/lcp/#measure-lcp-in-javascript.
+{
```

Comment:
This whole block should go into a static function in the Tracing Integration and called after setup once.

Reply:
It should also receive a gatekeep checking for the availability of performance API.

Reply:
I'm calling it in the constructor, same place where we check for performance API availability.